features/pakiti/client/configv3.pan

```diff
@@ -0,0 +1,37 @@
+unique template features/pakiti/client/configv3;
+
+variable PAKITI_RPMS ?= if_exists('config/pakiti/client/config');
```

Comment:
indeed metaconfig/pakiti configures the pakiti client v2. Should I upgrade that for v3 rather than the filecopy based config?